### ISSUE TITLE
"Distribute a privately hosted extension" does not work

### ISSUE THREAD

**Issue by pyBlob** [MicrosoftDocs/Edge-Enterprise#333]
We have followed all steps to pack and publish an extension to blob-storage for in-house use. When the extension is force-installed, the "edge://extensions/" page displays following error message, and the extension can not be enabled:
"Diese Erweiterung stammt nicht aus einer bekannten Quelle und wurde möglicherweise ohne ihr Wissen hinzugefügt."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d275477d-dfcc-30f8-431c-e73a346ae0c0
* Version Independent ID: 3f7c98ed-e7ff-1554-84dd-44319029900f
* Content: [Self-host Microsoft Edge extensions](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-manage-extensions-webstore)
* Content Source: [edgeenterprise/microsoft-edge-manage-extensions-webstore.md](https://github.com/MicrosoftDocs/Edge-Enterprise/blob/public/edgeenterprise/microsoft-edge-manage-extensions-webstore.md)
* Product: **microsoft-edge**
* GitHub Login: @AndreaLBarr
* Microsoft Alias: **aspoddar**

**Comment by Y-Kelly** [comment]
@pyBlob Can you share screenshots from edge://extensions/ and edge://policy/ pages?  Please also remove any private information before sharing.  Thanks!

**Comment by pyBlob** [comment]
Thanks for the fast response, I can provide you with more screenshots on Monday.

**Comment by pyBlob** [comment]
We have finally managed to install our extension without it getting blocked. At first, we have tried testing the install procedure using "Windows Sandbox". While the extension was visible, it could not be enabled because of:

[Microsoft Edge Browser Policy Documentation | Microsoft Docs](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies#extensioninstallforcelist)
> For Windows instances not joined to a Microsoft Active Directory domain, forced installation is limited to apps and extensions listed in the Microsoft Edge Add-ons website.

After using one of our spare company laptops, it worked.

This caveat should be more prominent, like the page linked in the document details.
